src/wrapper.jl

```diff
@@ -0,0 +1,138 @@
+wrap(base) = joinpath("wrapper", base * ".jl")
+
+# used by Clang.jl
+include(wrap("ctypes"))
+include(wrap("CEnum"))
```

Comment:
Is `CEum` copied from Clang.jl or did you write it? If it's copied then it needs a copyright notice.

Reply:
Are you suggesting to mention these files in the `LICENSE` file? If we add a note in the header, these would be overwritten by the wrapper generation script.

Reply:
I added a line to each of the files.